lxd/migrate_container.go

```diff
@@ -350,7 +350,7 @@ func (s *migrationSourceWs) Do(state *state.State, migrateOp *operations.Operati
 			return err
 		}
 
-		poolMigrationTypes = pool.MigrationTypes(storagePools.InstanceContentType(s.instance))
+		poolMigrationTypes = pool.MigrationTypes(storagePools.InstanceContentType(s.instance), false)
```

Comment:
@monstermunchkin hrm, so not familiar with the specifics here, but in your earlier commit messages about this change you mention that some storage drivers can change their preferred migration mechanism based on whether it is a refresh or not. As such, shouldn't the sending storage engine get the chance to alter their offered transport mechanisms based on whether it is a refresh or not? I'm thinking this should be dynamically set based on whether the migration is a refresh or not.

Reply:
I had code for that in place but removed it again. The reason being that it shouldn't matter to the sender what action is being performed. If the receiver only allows rsync due to a refresh, the sender will use the same migration type. At least that's my understanding of the code.
Also, the sender doesn't act differently if a refresh is being done.

Reply:
@monstermunchkin ok makes sense, I was just thinking in case in the future we need to influence the original offer header based on the action taking it would give the driver the opportunity to do that. Does it cause any harm passing in the refresh argument on the sender now, even if its not actually used?

Reply:
@tomponline No, it shouldn't cause any harm.

Reply:
@monstermunchkin in that case to keep things consistent it'd be good to include it on the offer side :) Thanks